frontend/components/CustomLink/CustomLink.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import Icon from "components/Icon";
+
+interface ICustomLinkProps {
+  url: string;
+  text: string;
+  /** Default opens in a new tab */
+  newTab?: boolean;
+  //* Icon wraps on new line with last word */
+  multiline?: boolean;
+}
+
+const baseClass = "custom-link";
+
+const CustomLink = ({
+  url,
+  text,
+  newTab = true,
```

Comment:
I do not think we should default to true for this prop. The component name itself is `CustomLink` component, so I would be confused if I was using this component and it automatically added an opening in new tab functionality. I think its more intuitive if worked closely to how `<a>` tags work, which require opt-in for new tab links.

Reply:
Wdyt about ExternalLink component because I don't want a new eng to jump in and not realize all external links open to a new tab. I have several doesnt external links I'm using this for and currently no in house links.

Reply:
I know we are currently using this component primarily for external links but there are links within the app that do not go to a new tab, we just have not updated them yet. If this is the case we would have a `CustomLink` and `ExternalLink`, the only difference being one shows an icon and opens in a new tab and the other doesn't.

That would be two places we have to keep the link text up to date and match if we needed/wanted to change it. I think having one linked component with a prop that changes if it opens in a new tab would be easier to maintain in the long run.

Reply:
I'm all about maintainability.